README.md

```diff
@@ -13,38 +13,104 @@ View the [list of supported APIs and Services](https://cloud.google.com/php/docs
 
 If you need support for other Google APIs, please check out the [Google APIs Client Library for PHP](https://github.com/google/google-api-php-client).
 
-## Quick Start
+## Quickstart
 
-We recommend installing individual component packages. A list of available packages can be found on [Packagist](https://packagist.org/search/?q=google%2Fcloud-).
+It is required to have the credentials configured to make requests to the api using the clients. In
```

Comment:
I am not sure this first sentence is very useful. I would suggest removing it entirely.
```suggestion
It is required to have the credentials configured to make requests to the api using the clients. In
```